src/rgw/rgw_crypt.cc

```diff
@@ -823,10 +823,7 @@ int RGWPutObj_BlockEncrypt::throttle_data(void *handle,
 
 std::string create_random_key_selector(CephContext * const cct) {
   char random[AES_256_KEYSIZE];
```

Comment:
This is an example of a case where I think something like
```
template<std::size_t N>
std::array<char, N> get_random_bytes()
```
would be useful.
```

Reply:
+1